sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Cluster Lifecycle Charter
+
+## Scope
+
+SIG Cluster Lifecycle’s objective is to improve the deployment, configuration management, and upgrade story for Kubernetes clusters and relevant infrastructure.
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Improving the Kubernetes UX for cluster administration
+- Kubernetes cluster lifecycle related features and issues
+- Tools that assist in bootstrapping, mutating, and upgrading Kubernetes control plane components. 
+- Kubernetes tools used to manage infrastructure related to provisioning and upgrading nodes
+- Tools that assist in management of configuration of Kubernetes components
```

Comment:
This was attempting to describe component configuration without saying component configuration (since that is the implementation of the goal, not the goal itself).

Reply:
Choosing sane defaults, reconfiguring, depreciation over time. 

Do you have a proposed change?